lms/djangoapps/grades/models.py

```diff
@@ -253,6 +253,14 @@ class Meta(object):
             # * Course staff can see all grades for a course using (course_id,)
             ('course_id', 'user_id', 'usage_key'),
         ]
+        # Allows querying in the following ways:
+        # (modified): find all the grades updated within a certain timespan
+        # (modified, course_id): find all the grades updated within a timespan for a certain course
+        # (modified, course_id, usage_key): find all the grades updated within a timespan for a subsection
+        #   in a course
+        index_together = [
+            ('modified', 'course_id', 'usage_key')
```

Comment:
At first glance, I was worried we had lost the index for just course_id, but we should be able to pull that off the unique index.